build-system/release-workflows/build-release.js

```diff
@@ -12,6 +12,13 @@ const jobName = 'build-release.js';
 
 runReleaseJob(jobName, async () => {
   const {ESM, FLAVOR} = process.env;
+
+  // TODO(danielrozenberg): remove temporary custom-config when ampjs.org
+  // becomes the default runtime CDN.
+  fs.writeJsonSync('build-system/global-configs/custom-config.json', {
+    'cdnUrl': 'https://ampjs.org',
+  });
+
   timedExecOrThrow(`amp release --flavor=${FLAVOR} --${ESM} --dedup_v0`);
```

Comment:
Can't you just set the env as part of this command?

Reply:
I don't think there's any part of the process that actually looks at the env variables. src/config.js has a variabled called `env` but it's [just an alias to `AMP_CONFIG`](https://github.com/ampproject/amphtml/blob/f564b7ee3cd59972fe86c4edc3052db70852c6f0/src/config.js#L8)

Reply:
Ahhhhhhhh.